python/ray/worker.py

```diff
@@ -668,6 +671,13 @@ def init(address=None,
     else:
         driver_mode = SCRIPT_MODE
 
+    if use_pickle:
```

Comment:
Ah I don't think this will cover the `ray start` case unless I'm missing something. Probably need to add it to `node.py` or `ray.connect()`

Reply:
Yeah I know, I'm still experimenting with this, it's more complicated than it should be :D